curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65552a9593755e1fb2f5ab50.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 65552a9593755e1fb2f5ab50
-title: Step 80
+title: Step 84
 challengeType: 20
-dashedName: step-80
+dashedName: step-84
 ---
 
 # --description--
```

Comment:
Modified instruction:
Define another function called `decrypt` with the same parameters as `encrypt`. This time return `vigenere(message, key, -1)`.